pulp_rpm/app/migrations/0030_DATA_fix_updaterecord.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 2.2.10 on 2020-02-19 14:43
+
+
+from django.db import models, migrations, transaction
+
+
+def replace_nonestring_with_null(apps, schema_editor):
+    with transaction.atomic():
+        UpdateRecord = apps.get_model('rpm', 'UpdateRecord')
+        for rec in UpdateRecord.objects.all().only('updated_date'):
+            if rec.updated_date in ('None', ''):
+                rec.updated_date = None
+                rec.save()
```

Comment:
Is it preferable over bulk_update?
Something like:
```
qs = UpdateRecord.objects.filter(updated_date__in=('None', ''))
UpdateRecord.objects.bulk_update(qs, ['updated'])
```
There are some caveats but I don't think we are affected, wdyt? 
https://docs.djangoproject.com/en/3.1/ref/models/querysets/#bulk-update
It's your call, I'm fine with any approach.